boards/raspberrypi/rpi_pico/rpi_pico-common.dtsi

```diff
@@ -94,6 +94,7 @@
 	status = "okay";
 	pinctrl-0 = <&uart0_default>;
 	pinctrl-names = "default";
+	fifo-enable;
```

Comment:
Does it make sense to have `fifo-enable;` defined in every board?

Perhaps it would be better to define it once in the SoC DeviceTree and use `/delete-property/ fifo-enable;` where it should be disabled. Or have `fifo-disable;` option instead.

Reply:
Generally, under the dts folder prefer "vanilla settings".
There are other devices with `fifo-enable` settings, but most are set under boards.